proposals/description/fip_76b.ts

```diff
@@ -0,0 +1,113 @@
+import { ProposalDescription } from '@custom-types/types';
+
+const fip_76b: ProposalDescription = {
+  title: 'FIP-76b: Add LUSD to FeiRari',
+  commands: [
+    {
+      target: 'delayedPCVMoverWethUniToBal',
+      values: '0',
+      method: 'unpause()',
+      arguments: [],
+      description: 'Unpause FEI/WETH delayed PCVMover'
+    },
+    {
+      target: 'delayedPCVMoverWethUniToBal',
+      values: '0',
+      method: 'withdrawRatio()',
+      arguments: [],
+      description: 'Finish migration of FEI/WETH to Balancer'
+    },
+    {
+      target: 'balancerDepositFeiWeth',
+      values: '0',
+      method: 'deposit()',
+      arguments: [],
+      description: 'Deposit WETH in Balancer'
+    },
+    {
+      target: 'aaveEthPCVDeposit',
+      values: '0',
+      method: 'withdraw(address,uint256)',
+      arguments: ['{uniswapPCVDeposit}', '400000000000000000000'],
+      description: 'Send 400 WETH to Uniswap PCVDeposit'
```

Comment:
Why are we adding 400 weth to the FEI/ETH uniswap pair?

Reply:
Compound uses UniswapV2AnchoredView so this will keep that pool liquid enough to not brick Compound